src/cljs/main/broadfcui/page/workspaces_list.cljs

```diff
@@ -204,28 +204,28 @@
                              :onClick #(swap! state assoc :show-access-level-select? true)}
                       "Include..."])
             :header-key "Include" :starting-width 68 :resizable? false :sort-by :none}]
-          :data (let [somepred (fn[preds]
-                                (->> (merge preds (:selected-types @state))
-                                     (keep (fn [[k v]] (when (and (not (false? v)) (some? (preds k))) k)))
-                                     (map preds)
-                                     (cons (constantly false)) ;; keeps (apply some-fn) from bombing when the list is empty
-                                     (apply some-fn)))]
+          :data (let [somepred (fn [preds]
+                                 (->> (merge preds (:selected-types @state))
+                                      (keep (fn [[k v]] (when (and (not (false? v)) (some? (preds k))) k)))
+                                      (map preds)
+                                      (cons (constantly false)) ;; keeps (apply some-fn) from bombing when the list is empty
+                                      (apply some-fn)))]
                   (filter
-                    (every-pred
-                      (somepred access-predicates)
-                      (somepred dataset-predicates)
-                      (somepred realm-predicates))
-                    (:workspaces props)))
+                   (every-pred
+                    (somepred access-predicates)
+                    (somepred dataset-predicates)
+                    (somepred realm-predicates))
+                   (:workspaces props)))
           :->row (fn [ws]
                    (let [ws-name (get-workspace-name-string ws)
-                         ws-href (let [x (ws "workspace")] (str (x "namespace") ":" (x "name")))
-                         disabled? (= (ws "accessLevel") "NO ACCESS")]
+                         ws-href (let [x (:workspace ws)] (str (:namespace x) ":" (:name x)))
```

Comment:
Another way of writing this `let` would be:
```clojure
(let [{:keys [namespace name]} (:workspace ws)] (str namespace ":" name))
```
Dubious whether or not this is actually clearer, I just like it when I can avoid naming new variables.